File: SDKs/twilio.com/twilio_conversations_v1/1.36.0/go/pkg/models/operations/listservice.go
package operations

import (
"openapi/pkg/models/shared")
var ListServiceServers = []string{
	"https://conversations.twilio.com",
}

type ListServiceQueryParams struct {
    PageSize *int64 `queryParam:"style=form,explode=true,name=PageSize"`
    
}

type ListServiceSecurity struct {
    AccountSidAuthToken shared.SchemeAccountSidAuthToken `security:"scheme,type=http,subtype=basic"`
    
}

type ListServiceRequest struct {
    ServerURL *string 
    QueryParams ListServiceQueryParams 
    Security ListServiceSecurity 
    
}

type ListServiceListServiceResponseMeta struct {
    FirstPageURL *string `json:"first_page_url,omitempty"`
    Key *string `json:"key,omitempty"`
    NextPageURL *string `json:"next_page_url,omitempty"`
    Page *int64 `json:"page,omitempty"`
    PageSize *int64 `json:"page_size,omitempty"`
    PreviousPageURL *string `json:"previous_page_url,omitempty"`
    URL *string `json:"url,omitempty"`
    
}

type ListServiceListServiceResponse struct {
    Meta *ListServiceListServiceResponseMeta `json:"meta,omitempty"`
    Services []shared.ConversationsV1Service `json:"services,omitempty"`
    
}

type ListServiceResponse struct {
    ContentType string 
    ListServiceResponse *ListServiceListServiceResponse 
    StatusCode int64 
    
}

File: SDKs/twilio.com/twilio_conversations_v1/1.37.3/python/src/sdk/models/operations/listservice.py
from dataclasses import dataclass, field
from typing import List,Optional
from dataclasses_json import dataclass_json
from sdk import utils
from sdk.models import shared


LIST_SERVICE_SERVERS = [
	"https://conversations.twilio.com",
]


@dataclass
class ListServiceQueryParams:
    page_size: Optional[int] = field(default=None, metadata={'query_param': { 'field_name': 'PageSize', 'style': 'form', 'explode': True }})
    

@dataclass
class ListServiceSecurity:
    account_sid_auth_token: shared.SchemeAccountSidAuthToken = field(metadata={'security': { 'scheme': True, 'type': 'http', 'sub_type': 'basic' }})
    

@dataclass_json
@dataclass
class ListServiceListServiceResponseMeta:
    first_page_url: Optional[str] = field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('first_page_url') }})
    key: Optional[str] = field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('key') }})
    next_page_url: Optional[str] = field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('next_page_url') }})
    page: Optional[int] = field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('page') }})
    page_size: Optional[int] = field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('page_size') }})
    previous_page_url: Optional[str] = field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('previous_page_url') }})
    url: Optional[str] = field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('url') }})
    

@dataclass_json
@dataclass
class ListServiceListServiceResponse:
    meta: Optional[ListServiceListServiceResponseMeta] = field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('meta') }})
    services: Optional[List[shared.ConversationsV1Service]] = field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('services') }})
    

@dataclass
class ListServiceRequest:
    query_params: ListServiceQueryParams = field()
    security: ListServiceSecurity = field()
    server_url: Optional[str] = field(default=None)
    

@dataclass
class ListServiceResponse:
    content_type: str = field()
    status_code: int = field()
    list_service_response: Optional[ListServiceListServiceResponse] = field(default=None)
    
